internal/serverinstall/nomad.go

```diff
@@ -432,6 +432,9 @@ func (i *NomadInstaller) Uninstall(ctx context.Context, opts *InstallOpts) error
 	s.Update("Removing Waypoint server from Nomad...")
 
 	_, _, err = client.Jobs().Deregister(serverName, i.config.serverPurge, &api.WriteOptions{})
+	if err != nil {
+		return err
```

Comment:
This one should also have a readable err message before returning the err, but something like:

> "Failed to deregister waypoint server"